x/incentives/keeper/distribute_test.go

```diff
@@ -181,6 +189,13 @@ func (suite *KeeperTestSuite) TestDistributeToConcentratedLiquidityPools() {
 			// We fix blocktime to ensure tests are deterministic
 			suite.Ctx = suite.Ctx.WithBlockTime(defaultGaugeStartTime)
 
+			// Set up authorized CL uptimes to robustly test distribution
+			if tc.authorizedUptimes != nil {
+				clParams := suite.App.ConcentratedLiquidityKeeper.GetParams(suite.Ctx)
+				clParams.AuthorizedUptimes = tc.authorizedUptimes
+				suite.App.ConcentratedLiquidityKeeper.SetParams(suite.Ctx, clParams)
+			}
+
```

Comment:
Every test case here defines authorizedUptimes as the supportedUptimes. Do you want to add some other test cases that do not touch this branch? If not / you do not see any value in it, we can just remove the param from the test case entirely and set it on every test run here.

Reply:
I had originally tabled this to avoid touching gauge logic per this issue #4985, but the change was small enough where I just tied it into this PR. Wider range of authorized uptimes (including default of 1ns) should be tested now.